### DISCUSSION TITLE
Refresher on how to use the Nissan e-NV

### DISCUSSION THREAD

**Discussion by hubdwoo** [Monash-Connected-Autonomous-Vehicle/autoware#35] [Announcements]
## Nissan e-NV startup

1. Unlock the car 
![car key](https://github.com/Monash-Connected-Autonomous-Vehicle/autoware/assets/36449214/80f89cc4-1801-44d3-ba64-f991812da2d5)

2. Turn on the car
2.1 Press on the brake
2.2 Press the ignition button
2.3 Make sure the "drive" logo is on
![drive logo](https://github.com/Monash-Connected-Autonomous-Vehicle/autoware/assets/36449214/29d34d94-28dd-4850-b988-1f56a4d600b6)

3. Turn on the computer battery
![battery pc](https://github.com/Monash-Connected-Autonomous-Vehicle/autoware/assets/36449214/bfea27e6-b022-4eba-9ab6-abd6774d9986)
4. End of use - make sure the computer battery is off and the car is locked to save charge on the 12 V battery. Because when the car is not locked there are some electronics that are dependent on the 12 V battery.

**Comment by dylan-gonzalez** [comment]
Thanks for the detailed instructions @hubdwoo !
